pkg/jobstore/boltdb/store_test.go

```diff
@@ -45,6 +45,7 @@ func (s *BoltJobstoreTestSuite) SetupTest() {
 
 	jobFixtures := []struct {
 		id              string
+		typ             string
```

Comment:
Stupid question - why not e the field "jobType" instead of "typ". Is this something idiomatic about go?

Reply:
type is a [keyword](https://go.dev/ref/spec#Keywords), but I guess jobType should be okay.